components/expandable/w-expandable.vue

```diff
@@ -31,12 +38,18 @@ if (!props.animated) {
   })
 }
 
+watch(expanded, (state) => {
+  setTimeout(() => {
+    showChevronUp.value = state
+  }, 200)
```

Comment:
Would it make sense to have the same comment on why 200ms, as you added in React?
```
 // We need a slight delay for the animation since it has a transition-duration of 150ms:

Reply:
Oh you're right! I'll add that! :)